frontend/src/components/question-creation/QuestionElement.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { HStack, Box } from "@chakra-ui/react";
+import { DeleteOutlineIcon, HamburgerMenuIcon } from "../../assets/icons";
+
+export interface QuestionElementProps {
+  children: React.ReactNode;
+}
+
+const QuestionElement = ({
+  children,
+}: QuestionElementProps): React.ReactElement => {
+  return (
+    <HStack spacing="6">
+      <Box color="grey.300">
+        <HamburgerMenuIcon />
```

Comment:
can we increase the size of these two icons?

Reply:
yes!